pkg/tfgen/docs.go

```diff
@@ -157,13 +157,39 @@ func boundedReplace(from, to string) tfbridge.DocsEdit {
 	}
 }
 
+func fixupImports() tfbridge.DocsEdit {
+
+	var inlineImportRegexp = regexp.MustCompile("% [tT]erraform import.*")
+	var quotedImportRegexp = regexp.MustCompile("`[tT]erraform import`")
+
+	// (?s) makes the '.' match newlines (in addition to everything else).
+	var blockImportRegexp = regexp.MustCompile("(?s)In [tT]erraform v[0-9]+\\.[0-9]+\\.[0-9]+ and later," +
+		" use an `import` block.*?```.+?```\n")
+
+	return tfbridge.DocsEdit{
+		Path: "*",
+		Edit: func(_ string, content []byte) ([]byte, error) {
+			// Strip import blocks
+			content = blockImportRegexp.ReplaceAllLiteral(content, nil)
+			content = inlineImportRegexp.ReplaceAllFunc(content, func(match []byte) []byte {
+				match = bytes.ReplaceAll(match, []byte("terraform"), []byte("pulumi"))
+				match = bytes.ReplaceAll(match, []byte("Terraform"), []byte("pulumi"))
```

Comment:
If TF is capitalized, the Pulumi should probably be capitalized too.
```suggestion
				match = bytes.ReplaceAll(match, []byte("Terraform"), []byte("Pulumi"))
```